Pages/lists.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists | OrcaUI</title>
+    <link rel="stylesheet" href="/Styles/OrcaUI-styles.css">
+    <link rel="stylesheet" href="/Styles/main.css">
+    <meta name="author" content="Hitesh Verma">
+    <meta name="Description" content="A variety of avatar to choose from.">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="/JS/toggle.js" defer></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="shortcut icon" href="/Assests/logo-rotated.svg" type="image/x-icon">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-T7RGFXZS8P"></script>
```

Comment:
you can create a separate js file or add script in the body rather than the head